src/main/java/seedu/address/commons/events/ui/LoginAppRequestEvent.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.logic.commands.LoginCommand;
+
+//@@author jelneo
```

Comment:
Names should not be included in the header comments as far as this iteration is concerned.

Reply:
Prof. Damith stated that we may start annotating our codes early in the slack channel.